website/client-old/js/services/userServices.js

```diff
@@ -247,11 +247,22 @@ angular.module('habitrpg')
             var tmp = res.data.data._tmp || {}; // used to notify drops, critical hits and other bonuses
             var crit = tmp.crit;
             var drop = tmp.drop;
+            var quest = tmp.quest;
 
             if (crit) {
               var critBonus = crit * 100 - 100;
               Notification.crit(critBonus);
             }
+
+            if (quest && user.party.quest) {
+              var userQuest = Content.quests[ user.party.quest.key ];
+              if (quest.progressDelta && userQuest.boss) {
+                Notification.quest('questDamage', quest.progressDelta.toPrecision(2));
```

Comment:
Let's match what's used in the marker in the menu bar. https://github.com/HabitRPG/habitrpg/blob/fefcb0f4aca56379a9734aec485ca9e8827c1ea0/website/client-old/js/controllers/menuCtrl.js#L56

We changed it from `toPrecision` to `toFixed` so that it wouldn't show as scientific notation.